curriculum/challenges/english/08-coding-interview-prep/rosetta-code/babbage-problem.english.md

```diff
@@ -6,17 +6,18 @@ challengeType: 5
 
 ## Description
 <section id='description'>
-<p><a href="https://en.wikipedia.org/wiki/Charles_Babbage" title="wp: Charles_Babbage">Charles Babbage</a>, looking ahead to the sorts of problems his Analytical Engine would be able to solve, gave this example:</p>
-<blockquote>What is the smallest positive integer whose square ends in the digits 269,696?</blockquote>
- <p> - Babbage, letter to Lord Bowden, 1837; see Hollingdale and Tootill, <i>Electronic Computers</i>, second edition, 1970, p. 125.</p>
-<p>He thought the answer might be 99,736, whose square is 9,947,269,696; but he couldn't be certain.</p>
-<p>The task is to find out if Babbage had the right answer.</p>
-<p>Implement a function to return the lowest integer that satisfies the Babbage problem. If Babbage was right, return Babbage's number.</p>
+<a href="https://en.wikipedia.org/wiki/Charles_Babbage" title="wp: Charles_Babbage" target='_blank'>Charles Babbage</a>, looking ahead to the sorts of problems his Analytical Engine would be able to solve, gave this example:
+<blockquote>
+  What is the smallest positive integer whose square ends in the digits 269,696?
```

Comment:
@scissorsneedfoodtoo What do you think about the indentation here?

Reply:
@NitronR, the indents here also don't matter, but I'll update the footer so there's a bit of a left margin.